File: Python/linear_search.py
# linear search function

def linear_search(arr, target):
    for i in range(len(arr)):
        if arr[i] == target:
            return i
    return -1


# Example
numbers = [1, 2, 3, 24, 56, 96]
target_element = 24

index = linear_search(numbers, target_element)

if index != -1:
    print(f"{target_element} found at index {index}")
else:
    print(f"{target_element} not found in the list")

File: Javascript/linearSearch.js
/**
 * Linear search algorithm to find an element in an array.
 * @param {Array} arr - The input array to search.
 * @param {any} target - The element to search for.
 * @returns {number} - The index of the target element in the array, or -1 if not found.
 */

function linearSearch(arr, target) {
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] === target) {
      return i; // Element found, return its index
    }
  }
  return -1; // Element not found
}

/* Example usage:
const arrayToSearch = [4, 2, 7, 1, 9, 5, 3];
const targetElement = 7;
const result = linearSearch(arrayToSearch, targetElement);

if (result !== -1) {
  console.log(`Element ${targetElement} found at index ${result}.`);
} else {
  console.log(`Element ${targetElement} not found in the array.`);
}
*/

// Export the function if you are working in a Node.js environment
// module.exports = linearSearch;
